lib/v2/bioone/featured.js

```diff
@@ -10,21 +11,21 @@ module.exports = async (ctx) => {
 
     const $ = cheerio.load(response.data);
 
-    const list = $('.items h4 a')
-        .slice(0, 10)
-        .map((_, item) => {
+    let items = $('.items h4 a')
+        .slice(0, ctx.params.limit ? parseInt(ctx.params.limit) : 10)
```

Comment:
`ctx.query.limit`. 
`ctx.params.limit` has no impact on the items count as it will be always 10 as `ctx.params.limit` is always undefined.